front/src/hooks/useLoadAndSetGlobalPreferences.ts

```diff
@@ -21,11 +21,8 @@ export const useLoadAndSetGlobalPreferences = () => {
 
```

Comment:
could you rename data in l.19 to something more descriptive, e.g. organisationAndBasesData